_stbt/color.py

```diff
@@ -0,0 +1,41 @@
+# coding: utf-8
+
+
+def get_color(region, frame=None):
+    """Get the average color of a region."""
+
+    import stbt
+
+    # Grab a frame if not provided by caller
+    frame = stbt.get_frame() if frame is None else frame
+
+    # Extract the requested region
+    frame = stbt.crop(frame.copy(), region)
+
+    # Convert the frame to a list and flatten it
+    frame = [item for sublist in frame.tolist() for item in sublist]
+
+    # Get total number of pixels
+    number_of_pixels = len(frame)
+
+    # Calculate the average color
+    blue = green = red = 0
+    for pixel in frame:
+        blue += pixel[0]
+        green += pixel[1]
+        red += pixel[2]
+
+    return blue/number_of_pixels, green/number_of_pixels, red/number_of_pixels
+
+
+def is_color(region, color_range, frame=None):
```

Comment:
I like that you give a minimum & maximum color. I think that we made a mistake in `stbt.ocr` by making `text_color` and `text_color_threshold`  parameters (i.e. `min = text_color - text_color_threshold` and `max = text_color + text_color_threshold`). Specifying the min & max directly is much easier; in practice I know the min & max that I want, and I have to calculate the average in order to specify `text_color` and `text_color_threshold`.